.github/workflows/mkdocs-dev.yaml

```diff
@@ -0,0 +1,34 @@
+name: Deploy the dev documentation
+on:
+  push:
+    paths:
+      - 'docs/**'
+      - mkdocs.yml
+    branches:
+      - main
+jobs:
+  deploy:
+    name: Deploy the dev documentation
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - name: Install dependencies
+        run: |
+          pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
+          pip install mike
+          pip install mkdocs-macros-plugin
+        env:
+          GH_TOKEN: ${{ secrets.ORG_GITHUB_TOKEN }}
+      - name: Configure the git user
+        run: |
+          git config user.name "ronaudinho"
+          git config user.email "doelaudi@gmail.com"
```

Comment:
might want to adjust this @itaysk

Reply:
can we just remove these lines? in other workflows where we git push without overriding the username, it shows as the "Github Actions" user, which is better I think